runtime/shared_common/CacheLifecycleManager.cpp

```diff
@@ -307,6 +308,11 @@ printSharedCache(void* element, void* param)
 		} else {
 			j9tty_printf(PORTLIB, "%-16s", "default");
 		}
+		if (currentItem->layer >= 0) {
+			j9tty_printf(PORTLIB, "%-12d", currentItem->layer);
```

Comment:
A width of 12 characters seems excessive for a value that is at most two digits, even the heading is only five characters.

Reply:
The output is:

```
Listing all caches in cacheDir /tmp/javasharedresources/

Cache name              level         cache-type      feature         layer       OS shmid       OS semid       last detach time

Compatible shared caches
Cache1                  Java8 64-bit  persistent      cr              0                                         Wed Aug 21 15:53:26 2019
```

It looks good to me. We can always change it later if someone complains about the space.

Reply:
I think we should make the output narrower, but I agree it can wait.